File: hencoder/30_lib_annotation/src/main/java/com/hsicen/a30_lib_annotation/BindView.java
package com.hsicen.a30_lib_annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/*** 注解*/
@Retention(RetentionPolicy.SOURCE)
@Target(ElementType.FIELD)
public @interface BindView {
	int value();
}

File: hencoder/30_annotation_reflection/src/main/java/com/hsicen/a30_annotation_reflection/BindView.kt
package com.hsicen.a30_annotation_reflection

/**
 * 作者：hsicen  2020/2/9 15:35
 * 邮箱：codinghuang@163.com
 * 作用：
 * 描述：注解
 */

@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.FIELD)
annotation class BindView(val value: Int)
